### DISCUSSION TITLE
Bug

### DISCUSSION THREAD

**Discussion by Gashout** [community/community#61222] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I tried all suggested steps in relevant issues including revoke of VSCode Github application. I was in beta and now I am in free trial mode. No need to share that this is not making me want to continue:

[INFO] [auth] [2023-07-19T08:25:41.259Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-07-19T08:25:41.261Z] GitHub Copilot could not connect to server. Extension activation failed: "No access to GitHub Copilot found. You are currently logged in as Gashout."

**Comment by KongSugoi** [comment]
You just log out ?

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @Gashout! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by davecheney** [comment, marked as answer]
@Gashout this is correct, there is no Copilot subscription for @Gashout
